crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/discover/test_get_cached_remotes_near_basis.rs

```diff
@@ -0,0 +1,128 @@
+use super::*;
+
+async fn mk_agent_info(u: u8) -> AgentInfoSigned {
+    AgentInfoSigned::sign(
+        Arc::new(KitsuneSpace::new(vec![0x11; 32])),
```

Comment:
There is a [function](https://github.com/holochain/holochain/pull/2656/files#diff-153b514261194dfafbb248fd4d704bd6c52b08292a8a27ca6bc588123715971aR60) that does this but it's not merged yet. It would be good to keep these test data builders outside tests so they can be re-used. 

I don't remember seeing something like this elsewhere but something to maybe move later if it's needed in other tests